src/status_im/contexts/wallet/wallet_connect/events.cljs

```diff
@@ -160,12 +163,23 @@
              :supported-namespaces supported-namespaces
```

Comment:
Unrelated to this PR, but the conversion of `supported-namespaces` to JS could be hidden away inside the effect so that we mostly play and worry about CLJ data.

Reply:
done